independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/catalog/Platform.java

```diff
@@ -14,6 +14,9 @@
 
     Map<String, Object> getMetadata();
 
+    @JsonIgnore
+    PlatformStream getStream(String id);
```

Comment:
I'd keep these interfaces w/o the Json annotations. There are corresponding JsonXXX implementations that are bound to Json. And I am not sure getters with arguments need this annotation anyway, do they?

Reply:
Ok removed all json annotation, new and existing